language/stdlib/modules/gas_schedule.mvir

```diff
@@ -244,9 +246,79 @@ module GasSchedule {
           Cost { cpu: 10, storage: 1 }
         );
 
+        // Native table. Do not change the order of these
```

Comment:
Should there be a comment here explaining what all these magic values are and why they're written out that way?

Reply:
Probably, but these are going to be moved into the genesis creation in rust in the next couple of days. I'll a brief comment though.